src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -270,6 +273,51 @@ private static bool GetSsl3Support()
             return (IsOSX || (IsLinux && OpenSslVersion < new Version(1, 0, 2) && !IsDebian));
         }
 
+        private static bool OpenSslGetTlsSupport(SslProtocols protocol)
+        {
+            if (!IsOpenSslSupported)
+            {
+                // on Windows and macOS TLS1.0/1.1 are supported.
+                return false;
```

Comment:
The comment doesn't make sense with the return.  Perhaps this if should just be `Debug.Assert(IsOpenSslSupported);`?